### ISSUE TITLE
Publish AWB on conda-forge

### ISSUE THREAD

**Issue by danielhollas** [aiidalab/aiidalab-widgets-base#595]
Just pasting here a handy guide how to create new conda-forge packages.

https://www.pyopensci.org/python-package-guide/tutorials/publish-conda-forge.html

**Comment by danielhollas** [comment]
@yakutovicha thanks. Happy to make a PR, but as I was looking through the code, I was wondering if this needs to be part of AWB at all, especially if we're going to make AWB only a library. 

Admittedly I know nothing about how this is used, but the code in `aiidalab_widgets_base/elns.py` does not use any of the other parts of AWB, and so it's completely independent. Should it be turned into its own library or app? Should it be part of `aiidalab-eln`?

**Comment by danielhollas** [comment]
I had two hours on the airport so I just decided to go for it. :sweat_smile: I published a new pre-release version which allowed me to generate a valid conda-forge recipe with grayskull. I've opened a PR on staged-recipes. https://github.com/conda-forge/staged-recipes/pull/26882

@yakutovicha @unkcpz could you please comment on that PR that you are okay with being added maintainers?

There is still some work left around dependencies, since `pip check` is failing after the conda install. Will continue to work on that.

**Comment by unkcpz** [comment]
> @yakutovicha @unkcpz could you please comment on that PR that you are okay with being added maintainers?

I am fine with it, thanks for adding me.

**Comment by danielhollas** [comment]
To clarify, you have to make a comment on the PR for staged-recipes, it's part of their checklist.

**Comment by unkcpz** [comment]
>  you have to make a comment on the PR for staged-recipes, it's part of their checklist.

So we are good now?

**Comment by danielhollas** [comment]
> So we are good now?

You need to make a comment on the PR :-P Here is the link https://github.com/conda-forge/staged-recipes/pull/26882

**Comment by unkcpz** [comment]
Ah, sorrry, I though this is the PR :smile:

**Comment by danielhollas** [comment]
The recipe has been merged and AWB is not on conda-forge! :tada: 
https://github.com/conda-forge/aiidalab-widgets-base-feedstock

For now @unkcpz  and I are the feedstock maintainers. It would be nice to have at least one more person. (@edan-bainglass LMK in case you have experience with conda-forge :-) )  

I've opened a testing PR for integrating AWB into the docker stack. https://github.com/aiidalab/aiidalab-docker-stack/pull/483

Closing this issue.
